mylyn.context/org.eclipse.mylyn.java.ui/src/org/eclipse/mylyn/internal/java/ui/editor/FocusedJavaAllProposalComputer.java

```diff
@@ -39,8 +39,12 @@ public FocusedJavaAllProposalComputer() {
 	@Override
 	public List computeCompletionProposals(ContentAssistInvocationContext context, IProgressMonitor monitor) {
 		if (shouldReturnResults()) {
-			List proposals = super.computeCompletionProposals(context, monitor);
-			return FocusedJavaProposalProcessor.getDefault().projectInterestModel(this, proposals);
+			try {
+				List proposals = super.computeCompletionProposals(context, monitor);
+				return FocusedJavaProposalProcessor.getDefault().projectInterestModel(this, proposals);
+			} catch (Exception e) {
+				return Collections.emptyList();
```

Comment:
I don't know when this exception occurs. Should we just ignore it or log as well?

Reply:
> I don't know when this exception occurs. Should we just ignore it or log as well?

This exception can occur with invalide java code, please see #370.
Example NPE in org.eclipse.jdt.internal.compiler.ast.*

So I think that logging this type of error is of no use.

Reply:
Thank you for confirming @BeckerFrank